MIGRATION.md

```diff
@@ -1,5 +1,13 @@
 # Migration
 
+## 4.1.0 → 5.0.0
```

Comment:
Please, add a note to CHANGELOG about the new version too.

Reply:
[Done](https://github.com/funbox/frontend-tests-runner/pull/20/commits/ff9f884390e7ee392bbc5a3f1f7fb42cf1b11883)

Reply:
Oh, wait. But the version of the package has not been changed.

Do you mind to bump it too? Set `version: 5.0.0` in package.json & package-lock.json.

Reply:
Thanks, fixed